core/dist/js/assets.json

```diff
@@ -41,11 +41,9 @@
   "../assets/plus-white.svg": "../assets/plus-white.svg",
   "../assets/video.svg": "../assets/video.svg",
   "decanter-grid.css": "../css/decanter-grid.css",
-  "decanter-grid.css.map": "../css/decanter-grid.css.map",
   "decanter-no-markup.js": "decanter-no-markup.js",
   "decanter-no-markup.js.map": "decanter-no-markup.js.map",
   "decanter.css": "../css/decanter.css",
-  "decanter.css.map": "../css/decanter.css.map",
```

Comment:
Is this right?

Reply:
Thats because we have been using the pretty format in this project up until now with does contrast with what we are doing in the Drupal world. I'm voting we go to the uglified version but that is another topic.